app/models/katello/glue/pulp/repo.rb

```diff
@@ -429,8 +429,8 @@ def puppet_module_ids
         Katello.pulp_server.extensions.repository.puppet_module_ids(self.pulp_id)
       end
 
-      def docker_image_ids
-        Katello.pulp_server.extensions.repository.docker_image_ids(self.pulp_id)
+      def docker_img_ids
```

Comment:
both of this and the rpm_ids were returning the actual ids of rpms or docker images from the database instead of running the method, I guess this is a change in rails 4?